### ISSUE TITLE
Pornhub download doesn't work (Text became null when parsing)

### ISSUE THREAD

**Issue by pingzoo** [AAndyProgram/SCrawler#145]
Tested URL:
https://pornhub.com/model/max-viksons
https://pornhub.com/model/kelvin-brent

Success-Popup, but no data downloaded.

LOGs:

<pre>
2023-06-16 02:52:54: [PornHub] - model_max-viksons: videos downloading error [https://www.pornhub.com/model/max-viksons/videos/upload]
	Text became null when parsing with PrePatterns(0)
	Parametername: Text
2023-06-16 02:52:54: Message: Text became null when parsing with PrePatterns(0)
	Parametername: Text
	HelpLink:
	StackTrace:    bei PersonalUtilities.Functions.UniversalFunctions.ErrorsDescriber.Execute(ErrorsDescriber e, Exception ex, MMessage Message, Object OptionalValueReturn)
	bei PersonalUtilities.Functions.RegularExpressions.RegexFunctions.RegexFields[T](String Text, IEnumerable`1 PrePatterns, IEnumerable`1 Fields, Object[] Arguments)
	bei SCrawler.API.PornHub.UserData.DownloadUserVideos(Int32 Page, CancellationToken Token)
	
	2023-06-16 01:43:12: [PornHub] - model_kelvin-brent: videos downloading error [https://www.pornhub.com/model/kelvin-brent/videos/upload]
	Text became null when parsing with PrePatterns(0)
	Parametername: Text
2023-06-16 01:43:12: Message: Text became null when parsing with PrePatterns(0)
	Parametername: Text
	HelpLink:
	StackTrace:    bei PersonalUtilities.Functions.UniversalFunctions.ErrorsDescriber.Execute(ErrorsDescriber e, Exception ex, MMessage Message, Object OptionalValueReturn)
	bei PersonalUtilities.Functions.RegularExpressions.RegexFunctions.RegexFields[T](String Text, IEnumerable`1 PrePatterns, IEnumerable`1 Fields, Object[] Arguments)
	bei SCrawler.API.PornHub.UserData.DownloadUserVideos(Int32 Page, CancellationToken Token)
</pre>

	Greetings
	Felix

**Comment by AAndyProgram** [comment]
**Not confirmed**. I don't have this error. Videos are downloaded.

Try to add cookies. Remember that PornHub is also [requires](https://github.com/AAndyProgram/SCrawler/wiki/Settings#pornhub-requirements) `cUrl` and `ffmpeg`. Please make sure that the correct paths to the programs are specified in the [settings](https://github.com/AAndyProgram/SCrawler/wiki/Settings#environment).

This is a screenshot that should prove that videos are downloading. I canceled the download when the first one was downloaded.
![image](https://github.com/AAndyProgram/SCrawler/assets/88590076/85cb0161-a7a4-434a-98f2-9e1f7e64469a)

Correct me if I'm wrong, but the title seems to be russian. Google autodetects this language in Google Translate when translating text `Утренняя дрочка`. Some users have already encountered an error when downloading with cyrillic characters in the path and/or file name. If so, I recommend changing the [cmd encoding](https://github.com/AAndyProgram/SCrawler/wiki/Settings#environment) to 1251.

----

Btw, cool profiles! 😊

**Comment by pingzoo** [comment]
Hmm now it works for me too. Maybe I did something wrong before. I think I just didn't notice that I should have scrolled down the list of cookies. Thanks for trying it on your end and sorry for the false alarm,

> Btw, cool profiles! 😊

Indeed! Especially Max is a dream :D

I need to stop this site-ripping-scraping-thing :) Since I got a little more familiar with PowerShell/Invoke-Webrequest and the Com-Object internetexplorer.application a few weeks ago I have already filled over 22 TB. :-)
